src/bidiTab/bidiTab.ts

```diff
@@ -68,14 +65,21 @@ const _waitSelfTargetIdPromise = _waitSelfTargetId();
 
   const bidiServer = await _createBidiServer(selfTargetId);
 
-  logSystem('launched');
+  bidiServer.getCommandProcessor().on('log', (...messages: unknown[]) => {
```

Comment:
let's not do this: all logs should be handled by the bidiServer.on('log', () => {}). If you need to differentiate between various types inside the bidiServer, let's include it as the first argument (this does not need to be type-safe I think so no cross-package imports are needed).

Reply:
even if I remove this, the browsing context log doesn't show up anywhere

(prior to this, it would show up in the console, as it had `console.log`)

Reply:
I have added a commit to this branch. PTAL.

Reply:
I think an alternative would be to always provide a logger function instead of relying on events. I think it makes things easier.

Reply:
Thanks. Browsing contexts seem to be still empty though:
<img width="933" alt="image" src="https://user-images.githubusercontent.com/2840106/218867308-3c916646-8176-42f0-a305-0dc831c9fb1d.png">

Reply:
I wonder if it could be a race condition, or just that we're setting up the listener too late (after the event is logged).

Reply:
You need to open a new context to see them I believe

Reply:
Ah, yes! 
<img width="893" alt="image" src="https://user-images.githubusercontent.com/2840106/218992617-47480f2a-d452-4306-b161-b5424b3936ba.png">

`browsingContext.create`.

However, we still do not see the _very first_ context. Is this OK?

Reply:
I am sure but I don't see a race condition: perhaps @sadym-chromium knows

Reply:
The context with the Mapper itself should not be shown there. We hide it from BiDi users (but not from CDP).

Reply:
Then it's WAI. Although, previously, the context with the mapper itself was being unconditionally logged. If hiding the very first context is not a regression then it's an improvement of the status quo.